mobile-app/app/screens/AppNavigator/screens/Loans/LoansScreen.tsx

```diff
@@ -128,28 +131,52 @@ export function LoansScreen ({ route }: Props): JSX.Element {
 
   // TODO: remove custom handling of empty vault display
   useEffect(() => {
-    if (route.params?.displayEmptyVault === undefined) {
-      setDisplayEmptyVault(true)
+    if (route.params?.loadingState === undefined) {
+      setLoadingState('empty_vault')
     } else {
-      setDisplayEmptyVault(route.params.displayEmptyVault)
+      setLoadingState(route.params.loadingState)
     }
-  }, [route.params?.displayEmptyVault])
+  }, [route.params?.loadingState])
 
-  if (displayEmptyVault === true) {
+   // TODO: remove fake loading of loans
+  useEffect(
+    () => {
+      const loansTimer = setTimeout(() => {
+        console.log('here')
+        setLoadingState('success')
+      }, 5000)
+
+      return () => {
+        clearTimeout(loansTimer)
+      }
+    }, [route.params?.loadingState])
+
+  if (loadingState === 'empty_vault') {
     return (
       <EmptyVault
         handleRefresh={() => {}}
         isLoading={false}
       />
     )
   }
+
   return (
     <ThemedView
       testID='loans_screen'
       style={tailwind('flex-1')}
     >
       <Tabs tabSections={tabsList} testID='loans_tabs' activeTabKey={activeTab} />
-      {activeTab === TabKey.YourVaults ? <Vaults /> : <LoanCards testID='loans_cards' loans={loans} />}
+      {activeTab === TabKey.YourVaults && <Vaults />}
+      {activeTab === TabKey.BrowseLoans && loadingState === 'loading'
+        ? (
+          <View style={tailwind('mt-1')}>
+            <SkeletonLoader
+              row={6}
+              screen={SkeletonLoaderScreen.Vault}
```

Comment:
This is for `Browse loans` tab right? Let's rename to `SkeletonLoaderScreen.Loan` and `LoansSkeletonLoader` to avoid confusion 😅